src/mscorlib/src/System/Math.cs

```diff
@@ -13,712 +13,73 @@
 
```

Comment:
The convention has been to rename the CoreCLR specific part like `Math.CoreCLR.cs`.

Reply:
Fixed.